warning NU1608: Detected package version outside of dependency constraint: AnalyzerTesting.CSharp.Extensions 1.2.0 requires Microsoft.CodeAnalysis.CSharp.Workspaces (= 4.3.1) but version Microsoft.CodeAnalysis.CSharp.Workspaces 4.13.0 was resolved.

Hi.

Thank you for RoslynAnalyzerTesting.CSharp.Extensions.

I get the following warning if my test project references the following NuGet packages:
```
<PackageReference Include="AnalyzerTesting.CSharp.Extensions" Version="1.2.0" />
<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.13.0" />
```

```
warning NU1608: Detected package version outside of dependency constraint: AnalyzerTesting.CSharp.Extensions 1.2.0 requires Microsoft.CodeAnalysis.CSharp.Workspaces (= 4.3.1) but version Microsoft.CodeAnalysis.CSharp.Workspaces 4.13.0 was resolved.
```

I believe it's caused by https://github.com/tom-englert/RoslynAnalyzerTesting.CSharp.Extensions/blob/cdb4defee7e1707179ea1a5ec2c26aa840a6405c/source/AnalyzerTesting.CSharp.Extensions/AnalyzerTesting.CSharp.Extensions.csproj#L15C5-L15C94

I think this:
`<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="[4.3.1]" />`
should be this:
`<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.1" />`

Thank you.

--- a/source/AnalyzerTesting.CSharp.Extensions/AnalyzerTesting.CSharp.Extensions.csproj
+++ b/source/AnalyzerTesting.CSharp.Extensions/AnalyzerTesting.CSharp.Extensions.csproj
@@ -12,7 +12,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing" Version="1.1.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing" Version="1.1.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing" Version="1.1.2" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="[4.3.1]" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.3.1" />
   </ItemGroup>
 
   <ItemGroup Label="Transitive Fixes">

